custom_components/google_home/models.py

```diff
@@ -37,6 +37,7 @@ def __init__(
         self.ip_address = ip_address
         self.hardware = hardware
         self.available = True
+        self.do_not_disturb = False
```

Comment:
Make it only available through getter.
```suggestion
        self._do_not_disturb = False
```